bigchaindb/web/websocket_server.py

```diff
@@ -60,7 +40,7 @@ def _multiprocessing_to_asyncio(in_queue, out_queue, loop):
 
     while True:
         value = in_queue.get()
-        loop.call_soon_threadsafe(_put_into_capped_queue, out_queue, value)
+        loop.call_soon_threadsafe(out_queue.put_nowait, value)
```

Comment:
The queue is unbounded, see:
 - https://github.com/bigchaindb/bigchaindb/pull/1403/files#diff-d0bb39e8d0b6c010c762ab9744cad775L164